README.md

```diff
@@ -1,32 +1,33 @@
 ## Gradle Thrift Plugin
 
-Gradle Thrift Plugin uses thrift compiler to compile Thrift IDL files.
+Gradle Thrift Plugin uses thrift compiler to compile Thrift IDL files. 
+This project is forked from [jruyi/thrift-gradle-plugin](https://github.com/jruyi/thrift-gradle-plugin)
```

Comment:
Maybe we can add the `Open-sourced by LINE Corporation` in the README like [Armeria](https://github.com/line/armeria/blob/main/README.md) and [Central Dogma](https://github.com/line/centraldogma/blob/main/README.md) projects?

```suggestion
This project is forked from [jruyi/thrift-gradle-plugin](https://github.com/jruyi/thrift-gradle-plugin)
It is open-sourced and licensed under [Apache License 2.0](https://www.tldrlegal.com/license/apache-license-2-0-apache-2-0) by [LINE Corporation](https://engineering.linecorp.com/en).
```

Note: Armeria and Central Dogma used different links for LINE Corporation:
* Armeria: https://engineering.linecorp.com/
* Central Dogma: https://linecorp.com/en/

I chose to use https://engineering.linecorp.com/ due to Github but I am OK with either one